iOS/README.md

```diff
@@ -0,0 +1,42 @@
+# PlayFab Multiplayer SDK for iOS
+
+## Logging
+
+The PlayFab Multiplayer C++ library includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed as an asset along with the application.
+
+When this file is detected by the application at runtime it will use it to enable logging as configured. The following verbosity levels are currently supported:
+1. `VERBOSE` - everything
+2. `INFO` - less than everything, only important messages and errors
+3. `ERROR` - only errors
+
+The logging is disabled by default, but can be enabled with "enabled" property set to `true`.
+
+### Example configuration file
+
+```json
+{
+    "enabled": false,
+    "bufferSize": 262144,
+    "maxNumberOfItemsInBatch": 500,
+    "maxBatchWaitTimeInSeconds": 1,
+    "readBufferWaitTimeInMilliseconds": 1,
+    "logFolder": "/app_sandbox_storage/Documents/",

```

Comment:
just curious
what is `app_sandbox_storage`?
Is this predefined absolute path or do we handle this as mapping app path?